src/components/HomeSectionClients/styled.ts

```diff
@@ -9,14 +9,31 @@ import fatLlamaUrl from './fat-llama.svg';
 import framerUrl from './framer.svg';
 import googleUrl from './google.svg';
 
-export const HomeLeftRightWrapper = styled(_HomeLeftRightWrapper)`
-  & + & {
-    margin-top: 36px;
-  }
+export const CasesItemsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
 `;
 
 const leftSpacing = 30;
 export const Content = styled.div`
+  flex-basis: 100%;
+  margin-bottom: 48px;
+  :nth-last-child(1) {
+    margin-bottom: 0px;
+  }
+  font-size: 16px;
```

Comment:
Nit: I’d reorder this to keep CSS declarations together – and separate from other rules:

```suggestion
  font-size: 16px;

  :nth-last-child(1) {
    margin-bottom: 0px;
  }
```

This is minor but this helps with readability.